src/NetworkPort_NetworkPort.php

```diff
@@ -98,6 +98,11 @@ public function getOppositeContact($ID)
      */
     public function createHub($netports_id, $entities_id = 0)
     {
+        $config = \Config::getConfigurationValues('inventory');
```

Comment:
Maybe can you add `$this->conf = $conf;` in `checkConf()`  method and just use `$this->conf` here?

Reply:
(goal would be to avoid extra db requests - do not forget to declare the `$conf` property in the class if you made that change)